Source/NSView.m

```diff
@@ -5212,6 +5211,30 @@ - (void) _setNeedsUpdateConstraints: (BOOL)needsUpdateConstraints
   _needsUpdateConstraints = needsUpdateConstraints;
 }
 
+- (void) _layoutViewAndSubViews
+{
+  if (_needsLayout)
+    {
+      [self layout];
+      _needsLayout = NO;
+    }
+
+  NSArray *subviews = [self subviews];
+  FOR_IN (NSView *, subview, subviews)
+    [subview _layoutViewAndSubViews];
+  END_FOR_IN (subviews);
+}
+
+- (GSAutoLayoutEngine*) _layoutEngine
+{
+  if (!([self window] && [[self window] _layoutEngine]))
```

Comment:
You don't need the second test. Just returning the value will be the same.

Reply:
Fixed